helpers/helpers.go

```diff
@@ -24,6 +25,22 @@ var (
 	pNTNUSD = decimal.RequireFromString("10.0")
 )
 
+func PrintUsage() {
```

Comment:
I think this can be auto-generated from the flag definitions

Reply:
resolved in 997d7f49c97fa012e19b509165f138c3b00391aa